<?xml version="1.0"?>
<launch>
  <!-- argument -->
  <arg name="robot_name" default=""/>
  <arg name="reset" default="false"/>
  <param name="reset_z" type="bool" value="true" if="$(arg reset)"/>
  <param name="reset_z" type="bool" value="false" unless="$(arg reset)"/>

  <!-- params -->
  <rosparam file="$(find whi_imu)/config/imu_hardware_jy61p.yaml" command="load" subst_value="true"/>

  <!-- hardware interface -->
  <node name="whi_imu" pkg="whi_imu" type="whi_imu_node" output="screen"/>
</launch>
